src/index.ejs

```diff
@@ -10,7 +10,7 @@
     script-src:
       - https://dap.digitalgov.gov will be inserted by htmlWebpackPlugin if env is configured with DAPTrackingURL. This hostname is needed for fetching and executing the DAP tracking script
     -->
-    <meta http-equiv="Content-Security-Policy" content="default-src 'self' https://login.bionimbus.org https://wayf.incommonfederation.org; child-src blob:; img-src 'self' https://opendata.datacommons.io https://static.planx-pla.net data: https://*.s3.amazonaws.com https://www.google-analytics.com blob:; script-src 'self' 'unsafe-eval' https://www.google-analytics.com <%= htmlWebpackPlugin.options.script_src %>; worker-src 'self' blob:; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com; object-src 'none'; font-src 'self' data: https://fonts.googleapis.com https://fonts.gstatic.com; connect-src 'self' https://login.bionimbus.org https://wayf.incommonfederation.org https://opendata.datacommons.io https://static.planx-pla.net https://www.google-analytics.com <%= htmlWebpackPlugin.options.connect_src %>; frame-src <%= htmlWebpackPlugin.options.connect_src %> 'self' https://auspice.planx-pla.net https://auspice.pandemicresponsecommons.org;;">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self' https://login.bionimbus.org https://wayf.incommonfederation.org; child-src blob:; img-src 'self' https://opendata.datacommons.io https://static.planx-pla.net data: https://*.s3.amazonaws.com https://www.google-analytics.com blob:; script-src 'self' 'unsafe-eval' https://www.google-analytics.com <%= htmlWebpackPlugin.options.script_src %>; worker-src 'self' blob:; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com; object-src 'none'; font-src 'self' data: https://fonts.googleapis.com https://fonts.gstatic.com; connect-src 'self' https://*.s3.amazonaws.com https://login.bionimbus.org https://wayf.incommonfederation.org https://opendata.datacommons.io https://static.planx-pla.net https://www.google-analytics.com <%= htmlWebpackPlugin.options.connect_src %>; frame-src <%= htmlWebpackPlugin.options.connect_src %> 'self' https://auspice.planx-pla.net https://auspice.pandemicresponsecommons.org;;">
```

Comment:
This was changed by https://github.com/uc-cdis/data-portal/pull/893 as requested by the security team, since putting `https://*.s3.amazonaws.com` in CSP was identified as a vulnerability by Veracode scan

You will have either to dynamically generate a more strict entry for this CSP, or work with the security team to figure out an alternative solution

Reply:
removed this and changed webpack.config.js to accept an array in process.env.DATA_UPLOAD_BUCKET , let me know if this solution would not work for you or if you would like any changes made